### DISCUSSION TITLE
Osu not starting on Fedora 37

### DISCUSSION THREAD

**Discussion by CoffeeIsLife87** [ppy/osu#21412] [Q&A] [answered]
`Failed to create CoreCLR, HRESULT: 0x8007054F` is the error it gives me when I run it in a terminal
No window ever shows and nothing appears in `dmesg` (some other thread said to check it so I figured I'd mention it)
No log files appeared in the location mentions in https://github.com/ppy/osu/discussions/15188
Lemme know if there's anything else I should check for diagnosing the issue

**Comment by CoffeeIsLife87** [comment, marked as answer]
update: This was the wrong spot to post it. I'm gonna move this to issues

**Reply by ghost** [reply]
iirc issues without known reproduction steps are thrown to discussions

looks like your new issue was also sent to discussions

**Comment by GoldM-amberdrake** [comment]
Might be the same issue as #21239 
Try downgrading libx11
